src/Platform/Microsoft.Testing.Platform/OutputDevice/TerminalOutputDevice.cs

```diff
@@ -213,39 +213,43 @@ public async Task DisplayBannerAsync(string? bannerMessage, CancellationToken ca
 
                 _bannerDisplayed = true;
 
-                if (bannerMessage is not null)
+                // Skip the banner when in server mode
+                if (!_isServerMode)
```

Comment:
There is already an `if (!_bannerDisplayed)` few lines above. Should we just change it to `if (!_bannerDisplayed && !_isServerMode)` ?
The difference in behavior is that we don't set `TESTINGPLATFORM_CONSOLEOUTPUTDEVICE_SKIP_BANNER`, which I think might be reasonable.

If I'm running a unit test that starts a child process in non-server mode, and I want to capture its standard output, I should be able to do that. We will probably hit this in our own integration tests, if any of our tests asserts the banner.